arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -556,31 +605,24 @@ bool BenchFeature::report(ClientFeature& client, std::vector<BenchRunResult> res
   printResult(output, builder);
   builder.close();
 
-  std::cout << "Min request time: " << std::setprecision(6)
+  std::cout << "Min request time: " << std::setprecision(4)
```

Comment:
is making results less precise intentional? why 6 digits was bad?

Reply:
We actually don't need more than 4 digits for handling the measurements in the histogram, it's less than milliseconds